payment/interfaces/offlinePaymentGateway.go

```diff
@@ -77,21 +77,37 @@ func (o *OfflineWebCartPaymentGateway) StartFlow(ctx context.Context, currentCar
 	return &domain.FlowResult{}, nil
 }
 
-//StartWebFlow for offline payment requires not much - just redirect to the returnUrl :-)
-func (o *OfflineWebCartPaymentGateway) StartWebFlow(ctx context.Context, currentCart *cartDomain.Cart, correlationID string, returnURL *url.URL) (web.Result, error) {
+// FlowStatus for offline payment is always finished
+func (o *OfflineWebCartPaymentGateway) FlowStatus(ctx context.Context, cart *cartDomain.Cart, correlationID string) (*domain.FlowStatus, error) {
+	return &domain.FlowStatus{
+		Status: "complete",
+		Action: "SHOW_SUCCESS_PAGE",
```

Comment:
I think this should return one of the newly introduced PaymentStatus constants, otherwise PaymentAction will handle default case. Unless this is intended

Reply:
fixed, thanks!